oktoberfest/preprocessing/preprocessing.py

```diff
@@ -301,7 +304,8 @@ def convert_search(
     else:
         raise ValueError(f"Unknown search engine provided: {search_engine}")
 
-    return search_result(input_path).generate_internal(tmt_labeled=tmt_label, out_path=output_file)
+    return search_result(input_path).generate_internal(tmt_labeled=tmt_label, out_path=output_file, 
```

Comment:
a few formatting issues here, please active pre-commit checks and ensure they run through to fix these things or execute `nox -rs pre-commit`